.env

```diff
@@ -0,0 +1,3 @@
+API_URL = "https://random-words-api-umber.vercel.app/word"
```

Comment:
Make a .env.sample or .env.dist, figure out what these are for and why it's good practice to have them in your code